pkg/cmdsetup/data.go

```diff
@@ -49,7 +50,8 @@ func newDataCmd(runners ...run.Unit) *cobra.Command {
 	metricSvc := observability.NewMetricService(metaSvc, metricsPipeline, "data", nil)
 	pm := protector.NewMemory(metricSvc)
 	pipeline := sub.NewServer(metricSvc)
-	propertySvc, err := property.NewService(metaSvc, pipeline, metricSvc, pm)
+	messenger := gossip.NewMessenger(metricSvc, metaSvc)
```

Comment:
Would you please move Messenger into the property package? It should be an embedded component of the property service. The gRPC port could be returned by the property service.